src/lib/config/nav-menu/tool-selector-nav-items.ts

```diff
@@ -47,30 +47,42 @@ export const toolSelectorNavItems = {
                             label: "Topcoder Academy",
                             url: "http://platform-ui.topcoder.com/learn",
                             icon: "tcacademy",
-                            description: "Hover description text"
+                            description: "Learn new skills"
                         },
                         {
                             label:"Dev Center",
                             icon: "dev-center",
                             url: "http://platform-ui.topcoder.com/dev-center",
-                            description: "Hover description text"
+                            description: "Get help for Topcoder tools"
+                        },
+                        {
+                            label:"Practice",
+                            // icon: "practice",
+                            url: "https://www.topcoder.com/challenges?search=practice&tracks%5BDS%5D=true&tracks%5BDes%5D=true&tracks%5BDev%5D=true&tracks%5BQA%5D=true&types%5B%5D=CH&types%5B%5D=F2F&types%5B%5D=TSK",
+                            description: "Learn to compete"
                         }
                     ]
                 },
                 {
                     label:"Compete",
                     children: [
                         {
-                            label: "MMs & RDMs",
-                            url: "",
+                            label: "Marathon Matches (MM)",
+                            url: "https://www.topcoder.com/challenges?search=marathon%20match&tracks%5BDS%5D=true&tracks%5BDes%5D=true&tracks%5BDev%5D=true&tracks%5BQA%5D=true&types%5B%5D=CH&types%5B%5D=F2F&types%5B%5D=TSK",
                             icon: "mm",
-                            description: "Hover description text"
+                            description: "Solve hard algorithm problems"
+                        },
+                        {
+                            label: "Rapid Dev Matches (RDM)",
+                            url: "https://www.topcoder.com/challenges?bucket=openForRegistration&search=Rapid%20Development%20Match&tracks%5BDS%5D=true&tracks%5BDes%5D=true&tracks%5BDev%5D=true&tracks%5BQA%5D=true",
+                            icon: "mm",
```

Comment:
Do we intend to use the same icon for RDM and MM?

Reply:
We don't have an icon for RDM in figma yet, this is just temporary, so we can update the Tool selector. Should I render the placeholder icon instead? 

I did this because RDM & MM were grouped together before.

Reply:
This is fine to use the same icon for now.